### ISSUE TITLE
past commits somehow being misplaced/undone

### ISSUE THREAD

**Issue by TallTed** [w3c/vc-data-model#837]
>> [@TallTed] The issue [I referenced](https://github.com/w3c/vc-data-model/pull/835#pullrequestreview-805415720) above was #777, which was addressed by PRs #785 and #786, but those no longer appear to have been merged? Unless maybe this PR is against the wrong base?
>
> [@msporny] Hrm, this PR is in the main history: https://github.com/w3c/vc-data-model/commit/a0f486365a2ab83a4bd67093c43b47a7198d129b but this one is not: https://github.com/w3c/vc-data-model/commit/e0ec8de9bcd0c809c78c09310ea201d3d3dc023f 
>
> @TallTed can you please open an issue to track this, we have to make sure we figure out what's going on here. Losing commits like this is a big deal/problem. The Editors are going to have to do some `git bisect` or `git blame` to figure out which PR overwrote those PRs. I'm noting some `git cherry-picking` going on and some main branch reverting, so its possible that there was some out of order application of the PRs.

_Originally posted by @msporny in https://github.com/w3c/vc-data-model/issues/835#issuecomment-970737539_

**Comment by kdenhartog** [comment]
I'll take a look at this one and report back my findings on this. It might have to happen this weekend depending on time available for the rest of the week.

From memory, these two areas were issues that commonly encountered the merge conflicts and could have been the case when we were rebasing things back together that we picked the wrong commit to accept. 

For this commit https://github.com/w3c/vc-data-model/commit/e0ec8de9bcd0c809c78c09310ea201d3d3dc023f in particular I had to manually migrate it over because it originally got merged onto the main branch instead of V1.1. Looking at PR #785 and #786 it looks like #785 was properly pointed at V1.1 where as #786 got pointed at main and we let it slip past us before merging. Seems like something related to that could have also been the issue at play here.

Thanks for raising this as an issue. I'll dig more into it and see what happened and also double check all the previous PRs we've merged didn't get accidentally merged to the wrong branch when I get the chance.

**Comment by kdenhartog** [comment]
Hey update on this. I didn't get time over the weekend to work on this but didn't forget about it. Hopefully I'll be able to get to it this week to close it out.

**Comment by kdenhartog** [comment]
From my investigations into this, [line 723 in index.html](https://github.com/w3c/vc-data-model/blob/e77940a0309cc43a1b198fc3e0769c449c25f796/index.html#L720) wasn't ever actually updated in #785. Instead it looks like this was a second instance of the same issue that we didn't manage to get updated at the same time.

It appears that #785 and #786 are correctly on V1.1 as well which is now our main branch. It also appears the changes from commit https://github.com/w3c/vc-data-model/commit/e0ec8de9bcd0c809c78c09310ea201d3d3dc023f have made it in via a rebased commit which can be found in V1.1 branch under the following commit https://github.com/w3c/vc-data-model/commit/41baf902955019441efbdf8ba158446eface280c which was added on October 1st from what I'm seeing in the git logs.

I've also manually checked all the PRs we've originally merged onto V1.2 or main branch (during the maintenance group so basically anything this year)have now successfully made it onto V1.1. 

@msporny @TallTed I feel confident that no issues remain with the V1.1 log and have opened a PR to address the separate text that still needs to be changed on line 723. That's in PR #841 and I believe once we can merge #841 we'll be good to close this issue.

**Comment by TallTed** [comment]
Thank you for doing that deep dive, @kdenhartog.  I will rely on your demonstrated eye for detail and accept that all PRs that ought now to be in the current branches, now are.

**Comment by msporny** [comment]
> @msporny @TallTed I feel confident that no issues remain with the V1.1 log and have opened a PR to address the separate text that still needs to be changed on line 723.

Thank you for the excellent sleuthing, @kdenhartog -- very much appreciated. :) 

Looks like we should focus on PR #841 to put this issue to rest.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-12-01-vcwg#section3-2) on 2021-12-01

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.2. past commits somehow being misplaced/undone (issue vc-data-model#837)

_See github issue [vc-data-model#837](https://github.com/w3c/vc-data-model/issues/837)._




**Brent Zundel:** I believe kyle did an update here..  

**Manu Sporny:** Ted, correct me if I'm wrong -- I think Kyle looked, and when he looked that almost all the text was updated except for that small chunk you found.  
… so, we did actually apply all the PRs properly. We just missed some of that text.  
… and I think Kyle raised a PR to remove that.  

**Ted Thibodeau Jr.:** my read was -- there was one commit that wasn't cherry-picked.  

**Brent Zundel:** once 841 is merged, that will be addressed..  

**Ted Thibodeau Jr.:** I think that's correct..  

</details>

**Comment by kdenhartog** [comment]
PR has been addressed in #841 - labelling this as errata and closing

**Comment by brentzundel** [comment]
I'm not sure it is appropriate to label this issue as `errata`. The intention of that label is to indicate that the issue describes an error found in the v1.0 data model recommendation. Is that the case here?

**Comment by msporny** [comment]
Removed Errata label -- mainly because the issue title is misleading -- we don't want something saying "past commits somehow being misplaced/undone" to show up in the errata document because, it turned out to not be true. 

This is a judgement call, but given that the issue was editorial in nature and had to do with fonts and readability, I don't think we need to list it in an errata document that's supposed to help people understand "dangerous mistakes that were made and fixed" (my interpretation of the purpose of an errata document).

Happy to add an errata label back in if folks feel strongly about this, but if we do, we need to change the issue title to something more representative of what happened like: "Fix readability/accessibility concerns related to font colors when describing diagrams."

**Comment by brentzundel** [comment]
The bad color text issue was #777 and is already in the errata list.

**Comment by kdenhartog** [comment]
👍 I think that assessment makes sense @msporny. I went the opposite direction just because I was hoping to make sure people were aware of this change having happened. Since this is really part 2 of the #777 issue as @brentzundel points out I can still achieve that by just linking to this issue and the PR in issue #777 and then we don't need the label here and everything is still findable.

**Comment by TallTed** [comment]
New issue title here should hopefully clarify matters for future reviewers...
